app/routes/capture-debt.js

```diff
@@ -7,10 +7,7 @@ const dateSchema = require('./schemas/date')
 const { getSchemeId, getSchemes } = require('../processing/scheme')
 const { convertToPence, convertDateToDDMMYYYY } = require('../processing/conversion')
 const { saveDebtData } = require('../processing/debt')
-const {
-  getPaymentRequestAwaitingEnrichmentWithValue,
-  getPaymentRequestAwaitingEnrichmentWithNetValue
-} = require('../payment-request/get-payment-request')
+const { getPaymentRequestAwaitingEnrichment } = require('../payment-request/get-payment-request')
```

Comment:
I'm wondering if there's a better way to organise all these consts - there's a lot of them

Reply:
I think there definitely is.  I've created an improvement ticket for this repo to better separate the concerns as there's quite a few file like this.  

If you look at the route handlers, they're doing too much in my opinion and if we break them down we would need less of these requires. 

So we can sort that as part of that ticket instead of this.

Reply:
Yep, sounds good :)